src/fides/cli/utils.py

```diff
@@ -117,15 +147,24 @@ def check_and_update_analytics_config(ctx: click.Context, config_path: str) -> N
             user={"analytics_opt_out": ctx.obj["CONFIG"].user.analytics_opt_out}
         )
 
-    is_analytics_opt_out = ctx.obj["CONFIG"].user.analytics_opt_out
```

Comment:
Is there a reason you replaced this temporary variable in the two places it was used in this function? Maybe your expecting it to change during the function itself?

Reply:
It was only defined here, and then used once. I removed it because it didn't seem useful and was just cluttering the function.